be/src/common/config.h

```diff
@@ -598,6 +598,8 @@ CONF_mInt32(zone_map_row_num_threshold, "20");
 //    Trace = 6
 CONF_Int32(aws_log_level, "3");
 
+CONF_mBool(enable_value_column_zonemap, "false");
```

Comment:
I think we can just remove this config, and just not add zone map for v1's value column, to make things simple.
Do we have any reason to add zonemap for v1's value column now?

Reply:
ok